apps/src/dance/ai/AiExplanationView.tsx

```diff
@@ -73,7 +73,7 @@ const AiExplanationView: React.FunctionComponent<AiExplanationViewProps> = ({
       data: CachedForegrounds as CachedWeightsMapping,
     },
     [FieldKey.BACKGROUND_PALETTE]: {
-      name: 'Background palette',
+      name: 'Background color',
```

Comment:
I wonder if it would make more sense to place Background color immediatey after Background effect.

Reply:
For sure.  This was a request from @samantha-code who can consider...